tools/mininet/Dockerfile

```diff
@@ -27,6 +27,9 @@ RUN PREFIX=/output make install-mnexec install-manpages
 RUN python3 setup.py install --root /output --prefix /usr/local
 # Install `m` utility so user can attach to a mininet host directly
 RUN cp util/m /output/bin/m && sed -i 's#sudo##g' /output/bin/m
```

Comment:
We already include the `m` in this image

Reply:
yes, we do, but this is an improved version of it, I have not removed the `m` script due to backwards compatibility with the tutorials.

Reply:
Sorry for the late reply, but I don't see any improvement in this change

`root@73ff3c4e4601:/bin# diff m host-cmd`

```diff
1a2,3
> # Copyright 2022-present Intel
> # SPDX-License-Identifier: Apache-2.0
43c45,46
< cmd="exec  mnexec $cg -a $pid $cmd"
---
> mnexec $cg -a $pid hostname $host
> cmd="exec mnexec $cg -a $pid $cmd"
root@73ff3c4e4601:/bin#
```